resume.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>wommy.io - resume</title>
+
+		<link rel='stylesheet' href='style.css'>
+		<style>
+			#main {
+				margin: 1rem;
+			}
+			h2 {
+				border-bottom: 1px solid silver;
+				padding-bottom: .3rem;
+			}
+			.margin{
+				margin: 1rem 0;
+			}
+			.indent{
+				padding-left: 1.25rem;
+			}
+			.highlight{
+				background-color: gainsboro;
+				border-radius: 3px;
+				padding: .2em .4em;
+				font-size: 85%;
+				font-family: monospace;
+			}
+		</style>
+	</head>
+	<body id='resume'>
+		<nav>
+			<a href='./index.html'>home</a>
+		</nav>
+		<div id='main'>
+			<h1 style="display:none;">resume</h1>
+			<div id='content'>
+				<h2 style="display:none;">contact</h2>
+					<div class='margin'>
+						<img src="./wommy.jpeg" alt="headshot" style="float:left; margin-right:20px;width:40%;">
+						<div><a href="./resume.pdf">download pdf</a></div>
+						<h2>TOMMY WILLIAMS</h2>
+						<div><span class='highlight'>email:</span> <a href='mailto:tom@technomad.media?subject=resume' target="_blank">tom@technomad.media</a></div>
+						<div><span class='highlight'>phone:</span> <a href='tel:3128189011' target="_blank">312 818 9011</a></div>
+						<div><span class='highlight'>portfolio:</span> <a href="github.com/wommy" target="_blank">github.com/wommy</a>, <a href="wommy.io" target="_blank">wommy.io</a>, <a href="technomad.media" target="_blank">technomad.media</a></div>
+					</div>
+					<div class='margin'>
+						<div><span class='highlight'>focus:</span> digital publishing, media distribution, ecommerce</div>
+						<div><span class='highlight'>location:</span> chicago, il; royal oak, mi; newyork, ny</div>
+						<div><span class='highlight'>hobbies:</span> cycling, martial arts, dancing, techno</div>
+						<div><span class='highlight'>built with:</span> linux, github, vscode, html, css</div>
+						<div><span class='highlight'>view online:</span> github.com/wommy/resume</div>
```

Comment:
Once you have the same class on a bunch of elements in a row, you know it's time to use a descendant selector in your css instead. Keep it DRY!

Reply:
if i were to keep it DRY, i would use a mixin
which i think would be better than `#contact div div span {` 

plus, later if need be,  i could kick up the highlight class to the global sheet

Reply:
Right but this is just html and css, so you don't have mixins. And using css selectors properly is better than repeating the same class over and over in your html.